nativelib/src/main/resources/scala-native/gc/commix/Phase.c

```diff
@@ -8,13 +8,13 @@
 
 void Phase_Init(Heap *heap, uint32_t initialBlockCount) {
     pid_t pid = getpid();
-    // size = static part + 32 bit int as string
-    char startWorkersName[32 + 10];
-    char startMasterName[31 + 10];
-    snprintf(startWorkersName, 32 + 10, "scalanative_commix_startWorkers_%d",
-             pid);
-    snprintf(startMasterName, 31 + 10, "scalanative_commix_startMaster_%d",
-             pid);
+    // size = static part + 20 digits (maximum 64 bit number) + null char
```

Comment:
Excellent!  Thank you for both taking the time and the helpful explanation.
Some devo down the line is going to sing your praises.

Reply:
Thanks! That's so kind of you